.github/workflows/tests.yml

```diff
@@ -107,3 +107,15 @@ jobs:
       run: |
         rm -rf "$(brew --repo homebrew/services)"
         ln -s "$PWD" "$(brew --repo homebrew/services)"
+
+  sorbet:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - name: Set up Homebrew
+      id: set-up-homebrew
+      uses: Homebrew/actions/setup-homebrew@master
+
+    - run: brew typecheck homebrew/services
```

Comment:
@issyl0 Think it'd be nice to put this into `brew test-bot --only-tap-syntax` for repos that have the necessary directories. Thoughts?

Reply:
Yes! I'll add that to my list.